notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-760_Analysis.ipynb

```diff
@@ -0,0 +1,441 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ab2599ca-375e-4b90-94af-3215bb826428",
+   "metadata": {},
+   "source": [
+    "## IMS mismatch with M1M3 motion system - SITCOM-760"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c80f073f-5eed-4280-991f-375e359e3644",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Define time frame to retrieve data in the EFD database\n",
+    "t_start = \"2023-05-30T04:35:00\"\n",
+    "t_end = \"2023-05-30T05:30:00\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b644b653-571d-4ee4-a013-ed825e16cf63",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "00bbb5e2-748e-493a-b3ec-fd4d8df4a45e",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import itertools as itt\n",
+    "import pandas as pd\n",
+    "import matplotlib.dates as mdates\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import seaborn as sns\n",
+    "\n",
+    "from astropy import units as u\n",
+    "from astropy.time import Time, TimezoneInfo\n",
+    "\n",
+    "from lsst.sitcom import vandv\n",
+    "from lsst.ts.idl.enums import MTM1M3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d933c046-a1a2-4d66-96a0-8a6b46bff479",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Create client to access the EFD database\n",
+    "client = vandv.efd.create_efd_client()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "361fe8dc-295b-4e01-a44a-363a8ab1b428",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "all_columns = [\"xPosition\", \"xRotation\", \"yPosition\", \"yRotation\", \"zPosition\", \"zRotation\"]\n",
```

Comment:
One can get the same with:
pos_columns = [a + "Position" for a in "xyz"]
rot_columns = [a + "Rotation" for a in "xyz"]
all_columns = pos_columns + rot_columns